packages/dm-core-plugins/src/table/TablePlugin.tsx

```diff
@@ -28,21 +28,23 @@ export const TablePlugin = (props: IUIPlugin) => {
   if (error) throw new Error(JSON.stringify(error, null, 2))
 
   return (
-    <Table
-      addItem={addItem}
-      config={config}
-      dirtyState={dirtyState}
-      idReference={idReference}
-      items={items}
-      isLoading={isLoading}
-      onOpen={props.onOpen}
-      reloadData={reloadData}
-      removeItem={removeItem}
-      saveTable={save}
-      setDirtyState={setDirtyState}
-      setItems={setItems}
-      updateItem={updateItem}
-      type={props.type}
-    />
+    <div className='dm-plugin-padding'>
```

Comment:
Maybe triple check that this maybe shouldn't have w-full as well - I think it works most places because we use flex pretty much everywhere that stretches children, but I don't know how this will look in grid if say the table is narrower in width than parent 🙂

Reply:
aha, I think that solves the table not being 100% on the botton I asked about earlier.